store/cockroach/cockroach.go

```diff
@@ -0,0 +1,113 @@
+package cockroach
+
+import (
+	"context"
+	"fmt"
+	"github.com/Masterminds/squirrel"
+	"github.com/cortezaproject/corteza-server/pkg/ql"
+	"github.com/cortezaproject/corteza-server/store"
+	"github.com/cortezaproject/corteza-server/store/rdbms"
+	"github.com/lib/pq"
+	"go.uber.org/zap"
+	"net/url"
+	"strings"
+)
+
+type (
+	Store struct {
+		*rdbms.Store
+	}
+)
+
+const (
+	storeTypeName = "cockroachdb"
+	// storeTypeWithDebug = "cockroachdb+debug"
```

Comment:
same as the comment below for the init() fn.

Reply:
Sure